litmus-portal/frontend/src/components/WelcomeModal/Stepper.tsx

```diff
@@ -0,0 +1,294 @@
+import Button from '@material-ui/core/Button';
+import Step from '@material-ui/core/Step';
+import { StepIconProps } from '@material-ui/core/StepIcon';
+import StepLabel from '@material-ui/core/StepLabel';
+import Stepper from '@material-ui/core/Stepper';
+import Typography from '@material-ui/core/Typography';
+import clsx from 'clsx';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import config from '../../config';
+import useActions from '../../redux/actions';
+import * as UserActions from '../../redux/actions/user';
+import { history } from '../../redux/configureStore';
+import { RootState } from '../../redux/reducers';
+import useStyles from './styles';
+import ModalPage from './Modalpage';
+import { Paper, TextField } from '@material-ui/core';
+import PasswordInput from '../PasswordInput/PasswordInput';
+import WelcomeModal from './index';
+function QontoStepIcon(props: StepIconProps) {
+  const classes = useStyles();
+  const { active, completed } = props;
+
+  return (
+    <div
+      className={clsx(classes.rootStepper, {
+        [classes.activeStepper]: active,
+      })}
+    >
+      {completed ? (
+        <div className={classes.circleStepper} />
+      ) : (
+        <div className={classes.circleStepper} />
+      )}
+    </div>
+  );
+}
+
+function CStepper() {
+  const classes = useStyles();
+  const user = useActions(UserActions);
+  const { userData } = useSelector((state: RootState) => state);
+  const [activeStep, setActiveStep] = React.useState(0);
+  const steps = ['', '', '', ''];
```

Comment:
You aren't setting the values of steps anywhere just initializing it to check the activeStep. You could do the same without using this and by checking the index values since it's just 4